src/developer-testing/dto/webhook.dto.ts

```diff
@@ -0,0 +1,18 @@
+/* istanbul ignore file */
+
+import { ApiProperty } from '@nestjs/swagger';
+import { IsString } from 'class-validator';
+
+export class WebHookDto {
+  @ApiProperty({
+    description: 'some text to send',
+    type: 'string',
+    example: '1',
+  })
+  @IsString()
+  readonly text: string;
+
+  constructor(dto: WebHookDto) {
```

Comment:
I think we don't specify constructors for dtos

Reply:
We do it for 8 other ones.

look for `constructor` while filtering files to `**/*.dto.*`

I am OK removing it, I think I copied an example with it included